packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -239,7 +262,8 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 	public ngOnInit(): void {
 		this.logger.debug(componentName + ": component initialized");
 		this.displayedColumns = [];
-		if (this.multiselect) {
+		if (this.isMultiSelectEnabled) {
```

Comment:
Now that you added some code for the `multiSelect` in the `ngOnChanges`, I think it is better to put this block also there, what do you think? Otherwise, this will only be executed once: when the component is initialized and not every time the `multiSelect` input changes

Reply:
True. I'll apply it :-)